client/auth/index.js

```diff
@@ -10,8 +10,9 @@ import config from 'config';
 import controller from './controller';
 
 export default () => {
+	page( '/oauth-login', controller.oauthLogin );
```

Comment:
If I understand correctly, we redirect on `client/boot/common.js`, so we redirect before we get here, so it should be ok to remove, no?
All the oauth stuff seems to be using the older Flux store, it might be a good opportunity to remove it ( maybe in a following PR )